dashboard/templates/small_multiples_files_bar_chart.html

```diff
@@ -9,11 +9,10 @@
     <script src="{% static 'underscore/underscore-min.js' %}"></script>
```

Comment:
This file can be removed from the repo as this is used for demo purposes

Reply:
Yeah that one and file_access_within_week.html used javascript resources from the web rather than locally, I switched that and this to use the local js files and seems like all the pages and tooltips worked fine.

Reply:
I wasn't able to test small_multiples as it didn't work anymore, it gives `GET http://localhost:5001/static/json/top_file_access_week_grade2.tsv 404 (Not Found)` I was going to ask you about that but am happy to just remove!